docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -9,13 +9,17 @@ Prerendering is an optimization technique that generates static HTML pages from
 
 To maximize the benefits of prerendering, target specific areas of your website that will gain the most from this optimization technique. Typically, these are high-traffic pages or sections featuring dynamic content that seldom changes. By focusing on these areas, you ensure efficient use of resources and faster load times for your users.
```

Comment:
Should we provide more guidance here?

Reply:
I am not sure that I get, so this selective prerendering can only be configured on CDN side not ours side.
also why this "sections featuring dynamic content that **seldom changes**."
Also we are not doing for users only for bots

Reply:
How should we update this section, then? Or should we remove it?

Reply:
I would say remove it